.github/workflows/apidiff.yaml

```diff
@@ -11,8 +11,22 @@ jobs:
     - uses: actions/checkout@v3
       with:
         ref: main
+    - uses: actions/setup-go@v3
+      with:
+        go-version: '1.18.3'
+    - name: Install latest apidiff
+      run: go install golang.org/x/exp/cmd/apidiff@latest
     - name: Create Go package baseline
-      run: apidiff -w pkg.main google.golang.org/genproto
+      run: apidiff -w pkg.main ./...
+    - name: Upload baseline package data
```

Comment:
I originally didn't have the upload/download steps, and it failed because it couldn't find the `pkg.main`. I believe it is because there is a second `checkout` step that checks out the PR branch to compare the `pkg.main` (created using `main` branch), and I think it either switches directories or wipes the working directory.

Reply:
Ah ic, you are right the second checkout wipes it out. In that case can we set the upload to only be live for 1 day with `retention-days`?

Reply:
Yes, good call.

Reply:
Done.